agi.go

```diff
@@ -138,7 +145,7 @@ func NewConn(conn net.Conn) *AGI {
 
 // NewStdio returns a new AGI session to stdin and stdout.
 func NewStdio() *AGI {
-	return New(os.Stdin, os.Stdout)
+	return NewWithEAGI(os.Stdin, os.NewFile(uintptr(3), "/dev/stdeagi"), os.Stdout)
```

Comment:
we cannot assume EAGI exists.  I'd make a new convenience wrapper function called NewEAGI() which does this.